genomic_data_service/region_indexer.py

```diff
@@ -274,9 +273,56 @@ def fetch_datasets(files, datasets):
         fetch_documents(dataset)
         datasets[dataset["accession"]] = dataset
 
+def get_encode_accessions():
```

Comment:
do we want to make an option to only index one assembly or the other?

Reply:
I can work on this when I come back from vacation.

Reply:
Is this one still on the table?

Reply:
Oh, right! I will work on it then get back to you. Thank you!